### PR TITLE
More Mission Descriptions

### PR DESCRIPTION
**Pull request by gettingusedto**
#### Summary
None

#### Purpose of change

Give more missions their description.
#### Describe the solution

Given descriptions to missions
- Recruit Tracker
- Recover Priest's Diary
- Investigate Cult
- Find Relic
- Prove your worth to foodperson
#### Describe alternatives you've considered

Not giving them description and let players interpret what the misssion name means.

#### Additional context

The description may need some corrections because im not that good in english. Any fixes to the description would be appreciated.

### CODE REVIEW

File: data/json/npcs/missiondef.json [hunk 1]
```diff
@@ -403,6 +404,7 @@
     "id": "MISSION_INVESTIGATE_CULT",
     "type": "mission_definition",
     "name": { "str": "Investigate Cult" },
+    "description": "Investigate the strange cabin in the woods and find an item that may clue in about <color_yellow>a potential human sacrifice</color>the cult has performed, such as <color_yellow>human bones and remains</color>.",
```
Comment (gettingusedto) [inline data/json/npcs/missiondef.json:407]:
```suggestion
    "description": "Investigate the strange cabin in the woods and find an item that may clue in about <color_yellow>a potential human sacrifice</color> the cult has performed, such as <color_yellow>human bones and remains</color>.",
```

File: data/json/npcs/missiondef.json [hunk 2]
```diff
@@ -403,6 +404,7 @@
     "id": "MISSION_INVESTIGATE_CULT",
     "type": "mission_definition",
     "name": { "str": "Investigate Cult" },
+    "description": "Investigate the strange cabin in the woods and find an item that may clue in about <color_yellow>a potential human sacrifice</color> the cult has performed, such as <color_yellow>human bones and remains</color>.",
```
Comment (DoctorBoomstick) [inline data/json/npcs/missiondef.json:407]:
```suggestion
    "description": "Investigate the strange cabin in the woods and find an item that may clue you in about <color_yellow>a potential human sacrifice</color> the cult has performed, such as <color_yellow>human bones or other remains</color>.",
```

File: data/json/npcs/missiondef.json [hunk 3]
```diff
@@ -698,6 +701,7 @@
     "id": "MISSION_RECRUIT_TRACKER",
     "type": "mission_definition",
     "name": { "str": "Recruit Tracker" },
+    "description": "Go to the cabin in the woods where the Tracker is camping at and <color_yellow>recruit them</color>.",
```
Comment (DoctorBoomstick) [inline data/json/npcs/missiondef.json:704]:
```suggestion
    "description": "Go to the cabin in the woods where the Tracker is camping and <color_yellow>recruit them</color>.",
```

**Review by DoctorBoomstick — CHANGES_REQUESTED** [review]

File: data/json/npcs/missiondef.json [hunk 4]
```diff
@@ -665,6 +667,7 @@
     "id": "MISSION_RECOVER_PRIEST_DIARY",
     "type": "mission_definition",
     "name": { "str": "Recover Priest's Diary" },
+    "description": "Find something that may reveal what the church's stance is on these events.  Something like <color_yellow>a priest's diary</color> will suffice.",
```
Comment (Karol1223) [inline data/json/npcs/missiondef.json:670]:
```suggestion
    "description": "Find something that may reveal what the church's stance is on these events.  Something like <color_yellow>a priest's diary</color> would suffice.",
```

File: data/json/npcs/missiondef.json [hunk 5]
```diff
@@ -776,6 +780,7 @@
     "id": "MISSION_PROVE_YOUR_WORTH",
     "type": "mission_definition",
     "name": { "str": "Prove your worth to Foodperson" },
+    "description": "Prove your dedication to Foodperson and Foodplace as a whole by finding <color_yellow>10 foodplace loyalty cards</color>.",
```
Comment (Karol1223) [inline data/json/npcs/missiondef.json:783]:
```suggestion
    "description": "Prove your dedication to the Foodperson and Foodplace as a whole by finding <color_yellow>10 foodplace loyalty cards</color>.",
```

File: data/json/npcs/missiondef.json [hunk 6]
```diff
@@ -287,6 +287,7 @@
     "id": "MISSION_GET_RELIC",
     "type": "mission_definition",
     "name": { "str": "Find Relic" },
+    "description": "Find a relic of a forgotten saint.  Places like a <color_yellow>cathedral or a museum</color> might have one.",
```
Comment (Karol1223) [inline data/json/npcs/missiondef.json:290]:
```suggestion
    "description": "Find a relic of a forgotten saint.  Places like <color_yellow>cathedrals</color> or <color_yellow>museums</color> might have one.",
```
I think I did the colors right on this one? But truth be told I've never done that before, so take this one with a grain of salt

**Pull request merged**